packages/stitch/tests/blah.test.ts

```diff
@@ -0,0 +1,327 @@
+import { parseSelectionSet } from '@graphql-tools/utils';
+
+const getFieldNode = (info: GraphQLResolveInfo) => {
+  if (info.fieldNodes.length !== 1) {
+    throw new Error('fieldNodes with length != 1 is not supported');
+  }
+
+  return info.fieldNodes[0];
+};
+
+const getOutputType = (schema: GraphQLSchema, name: string): GraphQLNamedOutputType => {
+  const type = schema.getType(name);
+  if (!type) {
+    throw new Error(`Could not get type ${name} from schema`);
+  }
+  if (!isOutputType(type)) {
+    throw new Error(`${name} is not an output type`);
+  }
+
+  return type;
+};
+
+describe('Blah', () => {
+  let aggSchema = makeExecutableSchema({
+    typeDefs: /* GraphQL */ `
+      type Query {
+        ccp: CcpQuery
+      }
+
+      type CcpQuery {
+        ccpEntitlement(id: ID!): CcpEntitlement
+        ccpProduct(id: ID!): CcpProduct
+      }
+
+      type CcpEntitlement {
+        id: ID!
+        offeringId: ID
+      }
+
+      type CcpProduct {
+        id: ID!
+        name: String
+      }
+    `,
+  });
+
+  let bbfSchema = makeExecutableSchema({
+    typeDefs: /* GraphQL */ `
+      type Query {
+        ccpOffering(id: ID!): CcpOffering
+      }
+
+      type CcpOffering {
+        id: ID!
+        productId: ID
+      }
+    `,
+  });
+
+  let stitchedSchema: GraphQLSchema;
+
+  let aggResolvers;
+
+  let bbfResolvers: any;
+
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  beforeEach(() => {
+    aggResolvers = {
+      CcpQuery: {
+        ccpEntitlement: async (_source: any, args: { id: string }) => {
+          return {
+            id: args.id,
+            offeringId: args.id,
+          };
+        },
+        ccpProduct: async (_source: any, args: { id: string }) => {
+          return {
+            id: args.id,
+            name: args.id,
+          };
+        },
+      },
+    };
+
+    bbfResolvers = {
+      Query: {
+        ccpOffering: async (_source: any, args: { id: string }) => {
+          return {
+            id: args.id,
+            productId: args.id,
+          };
+        },
+      },
+    };
+
+    aggSchema = addMocksToSchema({
+      schema: aggSchema,
+      resolvers: aggResolvers,
+    });
+    bbfSchema = addMocksToSchema({
+      schema: bbfSchema,
+      resolvers: bbfResolvers,
+    });
+
+    stitchedSchema = stitchSchemas({
+      subschemas: [{ schema: aggSchema }, { schema: bbfSchema }],
+      resolverValidationOptions: { requireResolversForResolveType: 'ignore' },
+      typeDefs: /* GraphQL */ `
+        extend type CcpEntitlement {
+          offering: CcpOffering
+        }
+
+        extend type CcpOffering {
+          product: CcpProduct
+        }
+      `,
+      resolvers: {
+        CcpEntitlement: {
+          offering: {
+            selectionSet: forwardArgsToSelectionSet('{ offeringId }'),
+            resolve: (parent, _args, context, info) => {
+              return delegateToSchema({
+                schema: bbfSchema,
```

Comment:
Delegate to a specific subschema, not to the stitched schema. I suspect that this is important to prevent infinite recursion, and why this test works without `skipTypeMerging: false`.